Documentation/Signoffs/Start Sensor/Start_Sensor_Detail_Design.md

```diff
@@ -0,0 +1,100 @@
+# Start Sensor System
+
+## Function of the Subsystem
+
+![Alt text](https://github.com/cebttu/CapstoneTeam1/blob/CazBilbrey-StartSensor-SignOff/Documentation/Signoffs/Start%20Sensor/Start%20Block%20Diagram.png)
+Figure 1: Start Sensor Block Diagram
+
+The Start Sensor System plays a pivotal role in launching the robot's predefined course. It employs an RGB color sensor programmed to identify a green LED, which marks the beginning of the robot's journey. Upon powering the sensor, if it detects the specified green color, it promptly communicates this information to the Main Controller, triggering the start state.
+
+To ensure operational robustness and mitigate the risk of sensor malfunction, a fail-safe option is in place. This involves a manual button that can be pressed in the event of sensor failure or unexpected issues. This redundancy enhances the robot's reliability and resilience in practical applications.
+
+
+## Constraints and Standards
+
+| Number | Constraint Description | Origin |
+|--------|------------------------|--------|
+| 1. | The RGB Sensor shall detect a spectrum of green light from an LED | Competition Specifications |
+| 2. | The RGB Sensor shall have an input of 4V to properly output | Sensor Specifications |
```

Comment:
What is the purpose of this as a constraint? As in, what does it help, prevent, or ensure?